poetry.lock

```diff
@@ -1752,25 +1749,7 @@ flake8-comprehensions = [
     {file = "flake8-comprehensions-3.8.0.tar.gz", hash = "sha256:8e108707637b1d13734f38e03435984f6b7854fa6b5a4e34f93e69534be8e521"},
     {file = "flake8_comprehensions-3.8.0-py3-none-any.whl", hash = "sha256:9406314803abe1193c064544ab14fdc43c58424c0882f6ff8a581eb73fc9bb58"},
 ]
-frozendict = [
```

Comment:
This shouldn't be removed; I think you have been bitten by the [recent poetry bug/interaction with PyPI](https://github.com/python-poetry/poetry/pull/5973). Can you do the following.

- check poetry version; should be poetry 1.1.14
- upgrade if necessary
- clear caches, see https://matrix-org.github.io/synapse/latest/development/dependencies.html#troubleshooting
- `poetry lock --no-update` (will be slow :cry: for the first run  after clearing caches)

which should get us back to a good state.

Reply:
Merci!